main/org.thingpedia.iot.battery/dataset.tt

```diff
@@ -0,0 +1,131 @@
+dataset @org.thingpedia.iot.battery language "en" {
+    program  := {
+  now => @org.thingpedia.iot.battery.state() => notify;
+}
```

Comment:
This is the old syntax. The new syntax is:
```
program := @org.thingpedia.iot.battery.state()
```

Did you run `make lint`? It should return a warning, and later Travis will fail because we're mixing new and old syntax.